locales/en.yml

```diff
@@ -124,6 +124,8 @@ en:
     copy_and_paste: 'Copy and paste code below into your CMS to embed your comment box in your articles'
     custom_css_url: 'Custom CSS URL'
     custom_css_url_desc: 'URL of a CSS stylesheet that will override default Embed Stream styles. Can be internal or external.'
+    custom_admin_css_url: 'Custom Admin Panel CSS URL'
```

Comment:
Wasn't sure how to indicate that other translations are needed.

Reply:
If a translation doesn't exist in the target language, English is used as the fallback. Typically, language authors just notice that change and resolve it, we're planning on some tooling in the future to help with this, but we have limited resources unfortunately.